### ISSUE TITLE
Config fulfillmentGasLimit for QRNG callback

### ISSUE THREAD

**Issue by EthanBlockson** [api3dao/airnode#1764]
We can [specify minConfirmations](https://forum.api3.org/t/qrng-minconfirmations/1872/6) to get Airnode callback faster.

```
        bytes32 requestId = airnodeRrp.makeFullRequest(
            airnode,
            endpointIdUint256,
            address(this),
            sponsorWallet,
            address(this),
            this.fulfillUint256.selector,
            abi.encode(
                bytes32("1s"),
                bytes32("_minConfirmations"),
                bytes32("10")
            )
        );
```

Does it possible to specify/set/config `fulfillmentGasLimit` same way? Or other way?

Because our logic needs up to 2-3M gas, and we run out of standard 500K.

**Comment by dcroote** [comment]
Thank you for the feedback. We are currently working on an approach to estimate gas, which will solve the issue: https://github.com/api3dao/airnode/issues/1589

**Comment by EthanBlockson** [comment]
Thank you. We have a couple of questions remaining. Could you please clarify them so that we can define the development path?

After this update (possibly `0.12.0`), if we use default airnode contract addresses like (`0x9d3C147cA16DB954873A498e0af5852AB39139f2` ANU QRNG or `0x224e030f03Cd3440D88BD78C9BF5Ed36458A1A25` Quintessence QRNG), which way can we make more gas-consuming calls?

1) They be able to handle callbacks with 2-3M gas by default
2) They can be modified directly inside calls using `setRequestParameters` or `makeFullRequest` `parameters`
3) New settings apply only to the individual `config.json` of self-hosted airnodes

**Comment by dcroote** [comment]
@EthanBlockson - functionality to estimate gas has been released in Airnode v0.12 as described in the second bullet of the [Release highlights](https://github.com/api3dao/airnode/releases/tag/v0.12.0).
